manifests/g/Google/Flutter/3.19.2/Google.Flutter.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=NVS2.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.Flutter
+PackageVersion: 3.19.2
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2024-02-28
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: flutter\bin\flutter.bat
```

Comment:
`.bat` files go against this repo's policy of not allowing scripts as an installer